apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedDevelopCRA.md

```diff
@@ -0,0 +1,20 @@
+UI Fabric also provides a starter using [Create React App](https://facebook.github.io/create-react-app/), a popular development stack maintained by the creators of React.
+
+Clone or fork this repo: https://github.com/kenotron/create-react-app-uifabric
+
+```shell
+git clone https://github.com/kenotron/create-react-app-uifabric.git my-app
```

Comment:
Are we OK with linking to a personal Github repository as part of our documentation?

Reply:
@kenotron do we have plans to move this into a microsoft repo sometime soon?

Reply:
Ken said he was working on moving them